src/App.js

```diff
@@ -1,11 +1,18 @@
-import './App.css';
+import styles from "App.module.css"
+import NavBar from "components/navBar"
+import TodoList from "components/todoList"
+import Header from "components/header"
 
 function App() {
   return (
-    <div className="App">
-     <h1>To do app</h1>
-    </div>
-  );
+    <main>
+      <NavBar />
+      <section className={styles.container}>
```

Comment:
By doing the above import we can do this. No need to use `styles.` everywhere

```suggestion
      <div className="container">
```